services/apis/fence/fenceTasks.js

```diff
@@ -217,17 +219,19 @@ module.exports = {
     }
 
     let res = await getNoRedirect(url, headers);
+    // console.log(`### NoRedirectURL Res: ${res.data}`);
```

Comment:
if this is useful in the future can you uncomment? otherwise I'd prefer to just completely remove the line (same for other debugging lines that are commented out)

Reply:
This is a left-over from the initial debugging.
We can actually access this information through the `--verbose` flag when we run `npm test`.

I'm happy to remove the line, however, I'd prefer to do it in a new PR 😄 (I want to merge this and monitor the results of other people's PRs -- I want to see how these changes behave out in the world and tweak / revert accordingly).